pkg/api/v1/summary_test.go

```diff
@@ -928,3 +928,324 @@ func TestListTabSummariesHTTP(t *testing.T) {
 		})
 	}
 }
+
+func TestListDashboardSummaries(t *testing.T) {
```

Comment:
nit: Looks good! Maybe a couple more test cases here and below:
- Dashboard group exists but has no dashboards
- Fetching correctly by normalized name
- Fetching correctly by normalized-equivalent name

Reply:
Good feedback! Will add those

Reply:
Done! PTAL - thanks :)